driver/k8s/logicalvolume_service.go

```diff
@@ -24,8 +25,13 @@ var ErrVolumeNotFound = errors.New("VolumeID is not found")
 
 // LogicalVolumeService represents service for LogicalVolume.
 type LogicalVolumeService struct {
-	client.Client
-	mu sync.Mutex
+	writer interface {
+		client.Writer
+		client.StatusClient
+	}
+	getter       *getter.RetryMissingGetter
```

Comment:
memo
`client.Reader` interface is not a match in this case. This is because `List` method is not used.
`List` method is difficult to realize read-after-create consistency behavior.